streaming/index.js

```diff
@@ -273,8 +311,10 @@ const startServer = async () => {
   // When checking metrics in the browser, the favicon is requested this
   // prevents the request from falling through to the API Router, which would
   // error for this endpoint:
+  // @ts-ignore
   app.get('/favicon.ico', (req, res) => res.status(404).end());
 
+  // @ts-ignore
   app.get('/api/v1/streaming/health', (req, res) => {
```

Comment:
These were due to unused parameters, we may not need those `@ts-ignore`s anymore.